app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/main/home/birthday/BirthdayViewModel.kt

```diff
@@ -170,7 +214,10 @@ class BirthdayViewModel(
         viewModelScope.launch(Dispatchers.IO) {
             birthdayRepository.deleteBirthday(id)
                 .onEach {
-                    it.onSuccess { navigateToHomeScreen() }
+                    it.onSuccess {
+                        navigateToHomeScreen()
```

Comment:
This maybe already fixed in other PR, but no need to pass this to `ViewModel`. Use flow instead.